src/components/Card/MyCard.styled.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+const StyledCard = styled.div`
+  width: 18rem;
+  border: 1px solid gray;
+  height: 45vh;
```

Comment:
Nie powinieneś definiowac konkretnej wysokości - to tekst powinien rozpychać element.
Zwróć uwage jak to jest robione w React Boostrap.